### DISCUSSION TITLE
Build keyboard! I do not get it !

### DISCUSSION THREAD

**Discussion by juankstylez** [nutgram/nutgram#294] [General]
Hi, I'm looking for a way to display this type of keyboard! I don't see it in the documentation! Can you tell me how I build it? thanks https://prnt.sc/nyqC-KDWdAUV

**Comment by Lukasss93** [comment]
Sorry! I'll update the doc soon adding a section for keyboards 😅

Anyway here an example:

![immagine](https://user-images.githubusercontent.com/4071613/212466392-67c2b9eb-a564-4059-b167-b468eefcb9fa.png)

Preview:
![Telegram_rzHQnY2egN-side](https://user-images.githubusercontent.com/4071613/212466528-8614e9e7-ce91-4ae9-8a2e-76bf03a5010c.png)
